<launch>
    <node name="audio_capture" pkg="audio_capture" type="audio_capture" required="true" output="screen">
      <param name="device" value="hw:tegrasndt19xmob,0"/>
      <param name="channels" value="1"/>
      <param name="sample_rate" value="48000"/>
      <param name="sample_format" value="S32LE"/>
      <param name="format" value="wave"/>
    </node>
</launch>